CHANGELOG.md

```diff
@@ -0,0 +1,14 @@
+# Amplify for Android (Beta) - CHANGELOG
+
+A declarative library for application development using cloud services.
+
+## 0.9.0
+
+Initial release! Includes Core features, plus support for these categories:
+
+- Analytics
+- API
+- DataStore
+- Hub
+- Logging
```

Comment:
Are these currently supported?

Reply:
We have plugins for those categories which customers can use.